### DISCUSSION TITLE
Sort 3rd click return to default

### DISCUSSION THREAD

**Discussion by vivianshub** [TanStack/table#2384] [General]
I have created a React Table with sorting enabled. When I click on a header once, it sorts ascending; i click the header a second time, it sorts descending. When I click  a third time, it sorts ascending. However, I want the third click to return to the defaultSorted. How would I go about doing this? I'm currently using react-table v6. Thanks!

**Comment by rossthedevigner** [comment]
@vivianshub I'm looking to do the same, did you ever figure it out?
